discovery/module.go

```diff
@@ -449,14 +462,40 @@ func (m *Module) Search(serviceID string, query map[string]string) ([]SearchResu
 			}
 		}
 
+		// extract registrationParameters from VP
+		registrationParameters := extractParameters(matchingVP)
+
 		result = append(result, SearchResult{
 			Presentation: matchingVP,
 			Fields:       fields,
+			Parameters:   registrationParameters,
 		})
 	}
 	return result, nil
 }
 
+func extractParameters(vp vc.VerifiablePresentation) map[string]interface{} {
```

Comment:
I'd not do this, and use the same principle with other extracted fields, using input descriptor field constraint IDs (like organization_name or ura)

Reply:
see other comment, using input descriptor field constraint IDs makes the PD too bloated.